### ISSUE TITLE
In.region returning inaccurate results

### ISSUE THREAD

**Issue by mluo123** [uclahs-cds/package-bedr#15]
When running the in.region function, the output often returns inaccurate results depending on the input format. 

Example input:
``
r1 <- c("chr2:70-100", "chr10:130-160");
``
``
r2 <- "chr2:80-90";
``

Code I used:
``
in.region(r1, r2);
``

Expected results:
``
[1] TRUE FALSE
``

Observed results:
``
[1] FALSE  TRUE
``

I found out that this issue was caused by these two lines in the in.region function:

``
x <- bedr.sort.region(x);
``
``
y <- bedr.sort.region(y);
``

Since bedr.sort.region sorts by 'lexicon' by default, this causes the output to return inaccurate results unless the input data is already sorted by lexicon. Having an option to sort by "natural" order depending on the input would be helpful

**Comment by tyamaguchi-ucla** [comment]
Good catch! How easy is it to add the option (probably adding a few lines of code)? It would be nice to submit a PR and add tests (see https://github.com/uclahs-cds/package-bedr/tree/main/tests/testthat). @dan-knight can provide guidance on adding unit tests.

**Comment by dan-knight** [comment]
> @dan-knight can provide guidance on adding unit tests.

Yep! This should (hopefully) be straightforward to test. It may also reveal some refactors that could be helpful.

**Comment by dan-knight** [comment]
Closed in #16
